File: B.Tech/SEM - 4/MADF/Extra/prime.dart
import 'dart:io';

void main(){
  stdout.write("N = ");
  int n = int.parse(stdin.readLineSync()!);

  bool flag = true;

  for(int i=2 ; i<n ;i++){
    if(n%i==0){
      flag = false;
      break;
    }
  }
  if(flag){
    stdout.write("$n is prime number.");
  }
  else{
    stdout.write("$n is not prime number.");
  }
}

File: B.Tech/SEM - 4/WT - II/Extra/prime.js
const prompt = require('prompt-sync')();

var n=prompt("Enter n : "), flag = true;


for(var i=2; i<n ; i++){
    if(n%i==0){
        flag = false
    }
}

if(flag)
    console.log(n," is prime number")
else 
    console.log(n," is not prime number") 
